src/general/OptionsMenu.tscn

```diff
@@ -34,6 +35,7 @@ __meta__ = {
 }
 
 [node name="Graphics" type="VBoxContainer" parent="TabContainer"]
+visible = false
```

Comment:
While this change doesn't change the tab behaviour this change to the visibility properties of the tabs should be undone.